src/NserviceBus.Transport.Msmq/MsmqTimeToBeReceivedOverrideCheck.cs

```diff
@@ -19,8 +19,19 @@ public StartupCheckResult CheckTimeToBeReceivedOverrides()
             var isTransactional = settings.GetRequiredTransactionModeForReceives() != TransportTransactionMode.None;
             var outBoxRunning = settings.IsFeatureActive(typeof(Features.Outbox));
 
-            var messageAuditingConfig = settings.GetOrDefault<AuditConfigReader.Result>();
-            var auditTTBROverridden = messageAuditingConfig?.TimeToBeReceived > TimeSpan.Zero;
+
+            //var messageAuditingConfig = settings.GetOrDefault<AuditConfigReader.Result>();
+            //var auditTTBROverridden = messageAuditingConfig?.TimeToBeReceived > TimeSpan.Zero;
+
+            //workaround for AuditConfigReader.Result not being accessible
```

Comment:
@Particular/nservicebus-maintainers `AuditConfigReader.Result` is internal, so this was a bit of a hack to get the info needed here. Maybe we should expose a way to get to the configured audit TTBR value?